dialect/sql/schema/legacy.go

```diff
@@ -0,0 +1,605 @@
+// Copyright 2019-present Facebook Inc. All rights reserved.
```

Comment:
Please, keep the previous name as I don't want to lose the git history (git blame) of this code. I use it in this project quite often.

Reply:
![image](https://user-images.githubusercontent.com/12862103/177188436-4d1486c7-365e-4d18-9803-ea64bf74b6f9.png)